src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/aktivitet/AktivitetController.kt

```diff
@@ -31,6 +31,6 @@ class AktivitetController(
         @PathVariable behandlingId: UUID,
     ): List<AktivitetArenaDto> {
         tilgangService.validerTilgangTilBehandling(behandlingId, AuditLoggerEvent.ACCESS)
-        return aktivitetService.hentAktivitetForBehandling(behandlingId)
+        return aktivitetService.hentAktiviteter(behandlingId)
```

Comment:
`hentAktiviteter` henter data for `fagsakPersonId`

Reply:
Burde denne endepunkten fjernes?  (kommentere med TODO) då jeg tolker det som at endepunkten for å hente vilkårperiode erstatter denne informasjonen

Reply:
Ja, skulle lagt på en kommentar 🤓  Denne skal fjernes/deprecates men lot den være for bakoverkompabilitet. Men skal fikse fagsakPersonId 👍